setups/index.ts

```diff
@@ -14,6 +14,14 @@ globalThis.fetch = fetch
 globalThis.Headers = Headers
 globalThis.Request = Request
 globalThis.Response = Response
+globalThis.URL.canParse = (url) => {
```

Comment:
```suggestion
globalThis.URL.canParse ??= (url) => {
```

Reply:
I can run tests on my machine. maybe we can upgrade our Node in the test environment instead of add this?

Reply:
Okay, the PR closed.